hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -1403,6 +1411,39 @@ public void testQuorumListenerConfig() {
         assertEquals("com.abc.my.second.listener", quorumConfig.getListenerConfigs().get(1).getClassName());
     }
 
+    @Test
+    public void testCacheConfig() {
```

Comment:
Here the same missing story. Documented as issue to add other attributes:
https://github.com/hazelcast/hazelcast/issues/12118